packages/lesswrong/components/messaging/NewConversationDialog.tsx

```diff
@@ -36,6 +36,8 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
   searchIcon: {
     marginLeft: 12,
+    color: theme.palette.grey[600],
```

Comment:
(Again not a change in this PR but I only just noticed it) - It seems a bit weird to me that the new conversation button has no hover style